components/OverviewCard/index.js

```diff
@@ -1,17 +1,28 @@
 import styled from "styled-components";
 import Link from "next/link";
 import { formatToEUR } from "@/helper/formatToEUR";
+import "chart.js/auto";
+import { Pie } from "react-chartjs-2";
+import { chartOptions } from "@/helper/chartOptions";
+import { createChartData } from "@/helper/createChartData";
+import useMedia from "@/hooks/useMedia";
+import { sumTotalPrice } from "@/helper/sumTotalPrice";
 
 export default function OverviewCard({ project }) {
-  const sumUpArray = (accumulator, currentValue) => accumulator + currentValue;
-  const totalPrice = project.items
-    .map((item) => Number(item.price))
-    .reduce(sumUpArray, 0);
-
+  const labelColor = useMedia(["(prefers-color-scheme: dark)"], [true], false)
+    ? "white"
+    : "black";
   return (
     <StyledProjectCard key={project.id}>
       <h2>Project: {project.name}</h2>
-      <p>Total: {formatToEUR(totalPrice)}</p>
+      <div>
```

Comment:
Do you really need this `div`? Without any styles or more than one child, it should work even without it.

Reply:
thanks for the reminder...was a left over